src/main/java/seedu/duke/Storage.java

```diff
@@ -209,16 +211,16 @@ public void loadPair(PairingList pairingList, File pairFile) {
                 String[] propertyParameters = pairingParameters[1].split("\\s\\|\\s");
 
                 //Client Information
-                String clientName = clientParamters[0];
+                String clientName = clientParamters[0].replace(OPEN_BRACKET, EMPTY_STRING);
```

Comment:
can change clientParamters to clientParameters to be more consistent with the others